airflow/utils/context.py

```diff
@@ -70,26 +66,21 @@
     "data_interval_start",
     "ds",
     "ds_nodash",
-    "execution_date",
     "expanded_ti_count",
     "exception",
     "inlets",
     "inlet_events",
     "logical_date",
     "macros",
     "map_index_template",
-    "next_ds",
-    "next_ds_nodash",
-    "next_execution_date",
+    "next_logical_date",
```

Comment:
Same

Reply:
I have removed: `prev_logical_date_success`, `prev_logical_date` and  `next_logical_date`.